include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -87,6 +91,7 @@ class BrushedMotorInterruptHandler {
     void homing_stopped() {
         hardware.reset_encoder_pulses();
         finish_current_move(AckMessageId::stopped_by_condition);
+        is_idle = true;
```

Comment:
Probably not, I was afraid that it would trip the timer interrupt to report a finish move without meeting the condition (if for whatever reason the encoder speed timer overflows right when homing_stopped happens), but i can test this to verify